draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1524,10 +1526,18 @@ Content-Type: text/plain
    as quoted-string (<xref target="quoted.strings"/>).
 </t>
 <t>
-   Because commas (",") are used as a generic delimiter between members of a
-   list-based field value, they need to be treated with care if they are
-   allowed as data within those members. Typically, list members that might contain a
-   comma are enclosed in a quoted-string.
+   A field's value can contain on or more members, separated by commas (",").
```

Comment:
This might require rephrasing, as not every occurence of "," inddicates multiple members.

Reply:
The paragraph directly below addresses this.

Reply:
I still think that the initial statement is misleading, if not even incorrect. Wil think about alternative text.